### PR TITLE
[Docs] Improve README

### PR DESCRIPTION
**Pull request by juntao**
Use book links when possible. Add new features we want to highlight.

Signed-off-by: Michael Yuan <michael@michaelyuan.com>

### CODE REVIEW

**Comment by alabulei1** [comment]
LGTM!

**Review by alabulei1 — DISMISSED** [review]
LTGM

File: README.md [hunk 1]
```diff
@@ -25,44 +25,44 @@ WasmEdge (previously known as SSVM) is a lightweight, high-performance, and exte
...
 * [Embed WasmEdge into a host application](https://wasmedge.org/book/en/embed.html)
 * [Orchestrate and manage WasmEdge instances using container tools](https://wasmedge.org/book/en/kubernetes.html)
+* [Run a WasmEdge app as a Dapr microservice](https://wasmedge.org/book/en/frameworks/mesh/dapr.html)
+* [Use Reactr to embed and extend WasmEdge functions in SaaS](https://wasmedge.org/book/en/frameworks/app/reactr.html)
 
 # Community
 
 ## Contributing
 
-If you would like to contribute to the WasmEdge project, please refer to our [CONTRIBUTING](docs/CONTRIBUTING.md) document for details. If you are looking for ideas, checkout our [wish list](docs/wish_list.md)!
```
Comment (hydai) [inline README.md:65]: Looks like we will no longer use this file `docs/wish_list.md`. Should we remove it directly?

**Review by hydai — CHANGES_REQUESTED** [review]

File: README.md (hunk 1 above)
Comment (juntao) [reply README.md:65]: I think we still leave that file in place because it might be linked to from articles we published on external websites.

Comment (hydai) [reply README.md:65]: How about adding a deprecated message on `wish_list.md` with a new link to our book?

Comment (juntao) [reply README.md:65]: I updated the wish list

**Review by hydai — APPROVED** [review]

**Pull request merged**